doc/source/conf.py

```diff
@@ -22,6 +22,7 @@
 extensions = ['sphinx.ext.autodoc', 'sphinx.ext.pngmath', 'numpydoc',
               'sphinx.ext.intersphinx', 'sphinx.ext.coverage',
               'sphinx.ext.doctest', 'sphinx.ext.autosummary',
+              'sphinx.ext.graphviz',
```

Comment:
The graphs depicting type hierarchy in `ufunc-overrides.rst`; e.g., https://github.com/numpy/numpy/pull/8247/files#diff-6aa7a114acf37c97c40cba1b3fe76900R261